_data/projects/SimpleLMS.yml

```diff
@@ -0,0 +1,12 @@
+name: SimpleLMS
+desc: Simple Learning Management System website script
+site: https://github.com/HikJ/SimpleLMS
+tags:
+- php
+- web
+- html
+- css
+- mysql
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/HikJ/SimpleLMS/issues
```

Comment:
Please constrain this link to the `up-for-grabs` tag (see others for link structure).